first_problem.rb

```diff
@@ -0,0 +1,16 @@
+class FirstProblem
+
+
+  def initialize(input_array)
+    @numbers_array = input_array
+  end
+
+  def compute()
+    sum = 0
+    @numbers_array.each do |number|
+      sum += number if number.even?
+    end
+    sum
```

Comment:
The `sum` lines in this code look really strange, are you familiar with the `Enumerator` concept? Can you express this as a concatenation of operations?